File: src/sdk-reference/cpp/1/auth/get-strategies/snippets/get-strategies.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  std::vector<std::string> strategies = kuzzle->auth->getStrategies();

  for (auto strategy: strategies) {
    std::cout << strategy << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/get-strategies/snippets/get-strategies.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  List<string> strategies = kuzzle.auth.getStrategies();

  foreach (var strategy in strategies) {
    Console.WriteLine(strategy);
  }
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}
